node/http.ts

```diff
@@ -247,7 +255,6 @@ export class Server extends EventEmitter {
           const req = new IncomingMessage(reqEvent.request);
           const res = new ServerResponse(reqEvent);
           this.emit("request", req, res);
-          this.#handler(req, res);
```

Comment:
I infer this is an optimization but it's not quite right, it causes observable differences between node and deno.